webpack.config.js

```diff
@@ -15,13 +17,11 @@ module.exports = {
     ],
   },
   output: {
-    // filename: '[name].bundle.js',
```

Comment:
hahah i see you played around a lot with this file. I appreciate your endless curiosity, Rick!! It's super admirable,  and I know it will make you a great engineer in future (well, you already one already 😄)